cypress/support/utils/shipper.js

```diff
@@ -0,0 +1,38 @@
+export class Shipper {
+  constructor(name) {
```

Comment:
I think that as long as we don't use arrays (eg. PurchaseOrderLine is the array belonging to PurchaseOrder) we can remove the constructor completely. (i don't think we have to init anything else except for arrays in these helper objects)

We aren't passing the name as parameter anyway when calling the constructor, so in cypress we will always see 2 log lines with `set name` except that the first one always has the value `undefined`.

Reply:
You are right, I'll change it.